app/javascript/styles/buttons.scss

```diff
@@ -0,0 +1,109 @@
+// Button styling.
+
+@import './bootstrap';
+@import './_variables';
+
+// btn-primary
+.btn-primary {
+  @extend .btn-primary;
+  background-color: $daria-color;
+  border: none;
+  color: $white;
+  margin-bottom: 2px;
+  font-size: $font-size;
+}
+
+.btn-primary:hover {
+  background-color: lighten($daria-color,15%);
+}
+
+.btn-primary:focus {
+  background-color: lighten($daria-color,15%);
+  color: $white;
+  text-decoration: none;
+}
+
+// btn-danger
+.btn-danger {
+  background-color: $dark-red !important;
+  border-color: $dark-red !important;
+}
+
+// btn-warning
+.btn-warning {
+  color: $black;
+  border-color: $daria-color;
+  background-color: $white;
+
+  &:hover {
+    background-color: $daria-color-lt;
+    border-color: $daria-color;
+  }
+  &:active {
+    background-color: $soft-blue !important;
+  }
+  &:focus {
+    background-color: $daria-color;
+    &:hover {
+      background-color: $daria-color-lt;
+    }
+  }
+}
+
+// btn-success
+.btn-success {
```

Comment:
Before: (right side, the yes button) ![image](https://user-images.githubusercontent.com/3866868/122292822-68143a00-cec4-11eb-9030-e67853a0c997.png)

After: TK

Reply:
After: 
![image](https://user-images.githubusercontent.com/3866868/122293778-5d0dd980-cec5-11eb-828d-daeb96da0454.png)